src/Elasticsearch.Net/Auditing/Auditable.cs

```diff
@@ -43,9 +42,11 @@ public string Path
 			set => _audit.Path = value;
 		}
 
+		public void Stop() => _audit.Ended = _dateTimeProvider.Now();
```

Comment:
LGTM, do we want this to be called multiple times to override or not? 

I think its fine as is.